com.unity.render-pipelines.high-definition/Editor/Lighting/SerializedHDLight.cs

```diff
@@ -496,5 +500,10 @@ void ApplyInternal(bool withDeportedEmissiveMeshData)
         }
 
         public void Apply() => ApplyInternal(withDeportedEmissiveMeshData: true);
+
+        public void Refresh()
+        {
+            throw new System.NotImplementedException();
```

Comment:
I'm assuming in the next step this Refresh will be used? Otherwise I'd question why it's in the interface if one of the two implementing classes doesn't need it...